src/index.js

```diff
@@ -63,13 +67,36 @@ function generateTravelerTripData(traveler) {
   })
 }
 
-function generateTripCosts() {
-  let soloTraveler = generateTraveler()
-  let yearOfTrips = soloTraveler.yearOfTrips()
-  let totalCosts = yearOfTrips.reduce((acc, trip) => {
+function findYearOfDestinations(traveler) {
+  let yearOfTrips = traveler.yearOfTrips()
+  let allDestinations = traveler.travelersDestinations
+  let thisYearsDestinations = [];
+  allDestinations.forEach(destination => {
+    yearOfTrips.forEach(trip => {
+      if (destination.id === trip.destinationID) {
+        thisYearsDestinations.push(destination)
+      }
+    })
+  })
+  return thisYearsDestinations
+}
 
+function generateTripCosts(traveler) {
+  let trips = traveler.yearOfTrips()
+  let destinations = findYearOfDestinations(traveler)
+  let totalSpent = destinations.reduce((acc, destination) => {
+    let lodging;
+    let lodgingAndFlight;
+    let totalCostsForAll;
+    trips.forEach(trip => {
+      lodging = (destination.estimatedLodgingCostPerDay * trip.duration)
+      lodgingAndFlight = (lodging + destination.estimatedFlightCostPerPerson)
+      totalCostsForAll = (lodgingAndFlight * trip.travelers)
+    })
```

Comment:
It might help, outside of this, if you iterated over trips and found the destination that has the matching id, then assigned it's values to said trip. You would then have all the necessary information in one data set. Then you wouldn't have to keep doing double iterators. I think this will also clean up some processing power and make it "faster" (idk if I'm using those terms correctly)

Reply:
Yea, it physically pained me to nest forEach(), but I'll have to see if I have time to refactor after I get to the MVP. Thank you!